h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3431,4 +3432,192 @@ def random_dataset_numeric_only(nrow, ncol, integerR=100, misFrac=0.01):
+                             nativeScoreTime, tolerance=1e-6):
+    '''
+    This method will summarize and compare H2OXGBoost and native XGBoost results for binomial classifiers.
+    This method will summarize and compare H2OXGBoost and native XGBoost results for binomial classifiers.
+
+    :param h2oPredictD:
+    :param nativePred:
+    :param h2oTrainTimeD:
+    :param nativeTrainTime:
+    :param h2oPredictTimeD:
+    :param nativeScoreTime:
+    :return:
+    '''
+    # Result comparison in terms of time
+    print("H2OXGBoost train time is {0}s.  Native XGBoost train time is {1}s.\n  H2OXGBoost scoring time is {2}s."
+          "  Native XGBoost scoring time is {3}s.".format(h2oTrainTimeD/1000.0, nativeTrainTime,
+                                                          h2oPredictTimeD, nativeScoreTime))
+    # Result comparison in terms of actual prediction value between the two
+    colnames = h2oPredictD.names
+    h2oPredictD['predict'] = h2oPredictD['predict'].asnumeric()
+    h2oPredictLocalD = h2oPredictD.as_data_frame(use_pandas=True, header=True)
+
+    # compare prediction probability and they should agree if they use the same seed
+    for ind in range(h2oPredictD.nrow):
+        assert abs(h2oPredictLocalD[colnames[2]][ind]-nativePred[ind])<tolerance, "H2O prediction prob: {0} and native " \
+                                                                         "XGBoost prediction prob: {1}.  They are " \
+                                                                         "very different.".format(h2oPredictLocalD[colnames[2]][ind], nativePred[ind])
+
+def summarizeResult_multinomial(h2oPredictD, nativePred, h2oTrainTimeD, nativeTrainTime, h2oPredictTimeD,
+                                nativeScoreTime, tolerance=1e-6):
+    # Result comparison in terms of time
+    print("H2OXGBoost train time is {0}s.  Native XGBoost train time is {1}s.\n  H2OGBoost scoring time is {2}s."
+          "  Native XGBoost scoring time is {3}s.".format(h2oTrainTimeD/1000.0, nativeTrainTime,
+                                                          h2oPredictTimeD, nativeScoreTime))
+    # Result comparison in terms of actual prediction value between the two
+    h2oPredictD['predict'] = h2oPredictD['predict'].asnumeric()
+    h2oPredictLocalD = h2oPredictD.as_data_frame(use_pandas=True, header=True)
+    nclass = len(nativePred[0])
+    colnames = h2oPredictD.names
+
+    # compare prediction probability and they should agree if they use the same seed
+    for ind in range(h2oPredictD.nrow):
+        for col in range(nclass):
+            assert abs(h2oPredictLocalD[colnames[col+1]][ind]-nativePred[ind][col])<tolerance, \
+                "H2O prediction prob: {0} and native XGBoost prediction prob: {1}.  They are very " \
+                "different.".format(h2oPredictLocalD[colnames[col+1]][ind], nativePred[ind][col])
+
+def genTrainFiles(nrow, ncol, enumCols=0, enumFactors=2, responseLevel=2, miscfrac=0):
```

Comment:
Minor: `getTrainFrames` ?

Reply:
That is correct, not a file!  I changed it to genTrainFrame.